pkg/endpoint/endpoint.go

```diff
@@ -2097,18 +2097,18 @@ func (e *Endpoint) Delete(monitor monitorOwner, ipam ipReleaser, manager endpoin
 		}
 	}
 
-	if option.Config.IPAM == option.IPAMENI {
+	if option.Config.IPAM == option.IPAMENI || option.Config.IPAM == option.IPAMAzure {
```

Comment:
v1.7 doesn't have Azure IPAM, did this backport cleanly?

Reply:
I marked the original PR as backport. But I wasn't aware we don't have Azure IPAM in v1.7.

Reply:
I thought it might be a concern when we merged it, but figured that you would backport it. I suspect I didn't actually voice this concern anywhere though :-)

We should likely drop all azure references from this backport commit.

Reply:
@jrajahalme There are still references to Azure IPAM in the code. Could you please remove those?

Reply:
Done. Someone had already reverted the line above, but there were some logs and comments that I reverted back to original.